src/blocks/file-blocks/code/index.tsx

```diff
@@ -1,39 +1,188 @@
 import "./style.css";
```

Comment:
you can probably nix this stylesheet

Reply:
actually, through a ridiculous amount of digging and trying things, you can do:

```
.cm-line {
  white-space: pre-wrap;
}
```

to wrap lines, which is really helpful for eg. the tutorial README file

Reply:
done!